Dockerfile

```diff
@@ -13,14 +29,19 @@ COPY CMakeLists.txt /root/history-tools
 COPY libraries /root/history-tools/libraries
 COPY src /root/history-tools/src
 COPY unittests /root/history-tools/unittests
+
 RUN mkdir /root/history-tools/build
 WORKDIR /root/history-tools/build
-RUN cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/postgresql ..
-RUN ninja && ctest --output-on-failure
+RUN cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/postgresql .. && \
+    ninja && \
+    ctest --output-on-failure
 
 FROM ubuntu:20.04
```

Comment:
Should this be a separate dockerfile, as it is not based on the other two in this file?

Reply:
Don't think so, I think is similar to the way followed in the fill-queue Dockerfile, I erased the line to separate those commands because it's dependant from the cmake copied in previous lines.

Reply:
I actually think this is fine. https://blog.alexellis.io/mutli-stage-docker-builds/